pandas/tests/series/accessors/test_cat_accessor.py

```diff
@@ -178,9 +180,9 @@ def test_dt_accessor_api_for_categorical(self):
         get_ops = lambda x: x._datetimelike_ops
 
         test_data = [
-            ("Datetime", get_ops(DatetimeIndex), s_dr, c_dr),
+            ("Datetime", get_ops(DatetimeArray), s_dr, c_dr),
```

Comment:
shouldn't the index accessors work as well here (eg the intent of the test is that)

Reply:
yah, its just the lists `DatetimeIndex._foo_ops` that we stop inheriting